### DISCUSSION TITLE
Best way to passing react props through three components

### DISCUSSION THREAD

**Discussion by maximhorse314** [maximhorse314/cg-frontend#3] [Q&A] [answered]
I have this kind of components tree in my react app. My header component includes the navbar component which in turn includes dialog1 and dialog2. So, I was wondering what is the best way to pass the props [opened1, opened2, handleDialog1, handleDialog2] to the dialog components from header (passing via navbar). Can you tell me the optimal solution?

**Comment by coozebra** [comment, marked as answer]
Personally I use a redux store, it can be implemented for three component but have much more possibilities Here you have a good article which explain how to use it
